modules/admin_manual/pages/enterprise/security/oidc/index.adoc

```diff
@@ -1,7 +1,6 @@
 = OpenID Connect (OIDC)
```

Comment:
You need to move this document (+navigation) out of enterprise as it is now GPL

Reply:
As suggested in #2100, we should put it somewhere around user management/SSO, likely here https://doc.owncloud.com/server/admin_manual/configuration/user/.

Reply:
Will move it.